packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
   * Migrate docs from terra-layout
   * Added docs for `terra-filter-pills`
   * Added tests for `terra-filter-pills`
+  * Migrate docs from terra-time-input
```

Comment:
This needs to be move up to the Unreleased section. The entry for terra-menu needs to move also.

Reply:
changed 1f38c2b4bd18f2546c492b916080c2f2b61b7917